### ISSUE TITLE
accessibility pass

### ISSUE THREAD

**Issue by matthewfdaniels** [the-pudding/climate-zones#7]

**Comment by matthewfdaniels** [comment]
- should be able to keyboard navigate any interactive elements
- anything relying on color (like city's climate category) should be available in alt/aria tags


see guidelines here:
https://www.notion.so/thepudding/Accessibility-Checklist-4cf13cb408454d9984e8c5fe88c518e2

**Comment by dktaylor916** [comment]
ive checked that you can scroll and tab through the elements for the interactievs and added the alt/aria tags
